src/valuetype_idtype.ts

```diff
@@ -71,7 +71,7 @@ async function guessIDType(def: ITypeDefinition, data: any[], accessor: (row: an
 
   const maxConfidence = Math.max(...confidences);
 
-  anyDef.idType = maxConfidence > 0.7? results[confidences.indexOf(maxConfidence)].idType : 'Custom';
+  anyDef.subType = maxConfidence > 0.7? results[confidences.indexOf(maxConfidence)].idType : 'Custom';
```

Comment:
why did you switch from `.idType` to `.subType` are you sure you changed all the occurrences where `.idType` was used?